zh_CN/changes/breaking-changes-5.5.md

```diff
@@ -0,0 +1,13 @@
+# EMQX 5.5 中的不兼容变更
```

Comment:
Does the change for MQTT Bridge V1 -> MQTT Bridge V2 need to be clarified?
@yanzhiemq

Reply:
yes, would you help to add this change? @wivwiv

Reply:
@Meggielqk please help to translate this breaking change to English